cmd/update-main.go

```diff
@@ -100,7 +100,7 @@ func GetCurrentReleaseTime() (releaseTime time.Time, err error) {
 
 func isDocker(cgroupFile string) (bool, error) {
 	cgroup, err := ioutil.ReadFile(cgroupFile)
-	if os.IsNotExist(err) {
+	if os.IsNotExist(err) || os.IsPermission(err) {
```

Comment:
As per my findings, `/proc/self/cgroup` in lxc/docker have permission like
```
-r--r--r-- 1 bala bala 0 Jun 12 07:25 /proc/self/cgroup
```
or
```
-r--r--r-- 1 root root 0 Jun 12 01:55 /proc/self/cgroup
```
I don't know how DC/OS does.

This means the file is always readable by non-root.   I think permission error should not occur, if so there is some setup problem which is hidden by this check.